bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingAddOp.java

```diff
@@ -456,7 +458,10 @@ private void maybeRecycle() {
     private void recyclePendAddOpObject() {
         entryId = LedgerHandle.INVALID_ENTRY_ID;
         currentLedgerLength = -1;
-        payload = null;
+        if (payload != null) {
+            ReferenceCountUtil.release(payload);
```

Comment:
`ReferenceCountUtil.safeRelease(payload);` would automatically do the if/null check